recipes/libgit2/meta.yaml

```diff
@@ -0,0 +1,165 @@
+            "git2/remote.h",
+            "git2/repository.h",
+            "git2/reset.h",
+            "git2/revert.h",
+            "git2/revparse.h",
+            "git2/revwalk.h",
+            "git2/signature.h",
+            "git2/stash.h",
+            "git2/status.h",
+            "git2/stdint.h",
+            "git2/strarray.h",
+            "git2/submodule.h",
+            "git2/sys/commit.h",
+            "git2/sys/config.h",
+            "git2/sys/diff.h",
+            "git2/sys/filter.h",
+            "git2/sys/hashsig.h",
+            "git2/sys/index.h",
+            "git2/sys/mempack.h",
+            "git2/sys/odb_backend.h",
+            "git2/sys/openssl.h",
+            "git2/sys/refdb_backend.h",
+            "git2/sys/reflog.h",
+            "git2/sys/refs.h",
+            "git2/sys/repository.h",
+            "git2/sys/stream.h",
+            "git2/sys/transport.h",
+            "git2/tag.h",
+            "git2/trace.h",
+            "git2/transaction.h",
+            "git2/transport.h",
+            "git2/tree.h",
+            "git2/types.h",
+            "git2/version.h"
+    ] %}
+
+    # Linux libraries.
+    {% set linux_libs = [
+            "libgit2.so",
+            "libgit2.so." + version,
+            "libgit2.so." + version_minor
+    ] %}
+
+    # OSX libraries.
+    {% set osx_libs = [
+            "libgit2.dylib",
+            "libgit2." + version + ".dylib",
+            "libgit2." + version_minor + ".dylib"
+    ] %}
+
+    # Windows libraries.
+    {% set win_libs = [
+            "%LIBRARY_BIN%\\git2.dll",
+            "%LIBRARY_LIB%\\git2.lib"
+    ] %}
+
+    {% for include in includes %}
+    - test -f $PREFIX/include/{{ include }}                 # [not win]
+    - if not exist %LIBRARY_INC%\\{{ include }} exit 1       # [win]
+    {% endfor %}
+
+    {% set libs =  linux_libs %}                            # [linux]
+    {% set libs =    osx_libs %}                            # [osx]
+    {% set libs =    win_libs %}                            # [win]
+    
+    {% for lib in libs %}
+    - test -f $PREFIX/lib/{{ lib }}                         # [not win]
+    - if not exist {{ lib }} exit 1                         # [win]
+    {% endfor %}
+
+about:
+  home: https://libgit2.github.com
+  license: GPLv2 with Linking Exception
+  license_file: COPYING
+  summary: 'libgit2 is a portable, pure C implementation of the Git core methods provided as a re-entrant linkable library with a solid API, allowing you to write native speed custom Git applications in any language which supports C bindings.'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    libgit2 is a portable, pure C implementation of the Git core methods 
+    provided as a re-entrant linkable library with a solid API, allowing 
+    you to write native speed custom Git applications in any language 
+    which supports C bindings.
+
+extra:
+  recipe-maintainers:
+    - shadowwalkersb
+    - jakirkham
```

Comment:
Good to hear it may be useful! Next one I have in mind is [pygit2](https://github.com/libgit2/pygit2). But, probably, after some other priorities.